java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -197,6 +199,9 @@ export function activate(context: ExtensionContext) {
         client.start();
         client.onReady().then((value) => {
             commands.executeCommand('setContext', 'nbJavaLSReady', true);
+            client.onNotification(StatusMessageRequest.type, (mp) => {
```

Comment:
`client.onNotification(StatusMessageRequest.type, showStatusBarMessage)` would be shorter. Shall this registration really be done in `onReady` callback, shall not it be done sooner without any callback?

Reply:
I've read in the documentation that the registration shall be done after onReady(), Will shorten the call.